docs/topics/con_server-sent-events.adoc

```diff
@@ -0,0 +1,16 @@
+[id="server-sent-events_{context}"]
+= Server-sent events
+
+Server-sent events (SSE) is a reactive HTTP-based messaging protocol.
+SSE works by establishing a connection between the event source and the client, and allowing the event source, such as an application server or a messaging broker, to use this connection to push events to the client when they occur.
+The connection remains open after the updates are pushed, and can be re-used to push subsequent updates.
+SSE represents a more resource-efficient alternative to polling, where a new connection must be established each time the client polls the events source for updates.
+As opposed to WebSockets, SSE only pushes events in one direction (that is, from the source to the client).
+It does not handle bidirectional communication between the event source and the client.
+
+The specification for SSE is incorporated into HTML5, and is widely supported by web browsers, including their legacy versions.
+SSE can be used from the command line, and is relatively simple to set up compared to other protocols.
+
+SSE is suitable for use cases that require frequent updates from the server to the client, while updates from the client side to the server are expected to be less frequent.
+Updates form the client side to the server can then be handled handled over a different protocol, such as REST.
```

Comment:
handled handled: Remove the extra word

Reply:
Thank you for catching this, I totally missed it:)